api/tests/v2/test_email.py

```diff
@@ -0,0 +1,127 @@
+from django.test import TestCase
+from rest_framework.test import APIRequestFactory, force_authenticate
+import mock
+
+from api.v2.views import (
+    VolumeSupportEmailViewSet, InstanceSupportEmailViewSet,
+    FeedbackEmailViewSet, ResourceEmailViewSet
+)
+from api.tests.factories import (
+    AnonymousUserFactory, InstanceFactory, UserFactory, VolumeFactory
+)
+
+
+class EmailTestHelper(TestCase):
+    """
+    This class is used for testing the following ViewSets:
```

Comment:
Other small note. A class shouldn't mention who uses it, that is bound to become out of date, but more importantly the class doc should just describe what the class itself is responsible for